test/cmdlineTests/yul_verbatim/input.yul

```diff
@@ -0,0 +1,11 @@
+{
+    let x := 2
+    let y := sub(x, 2)
+    let t := verbatim_2i_1o("abc", x, y)
```

Comment:
Since in other tests `hex""` is used, I'd prefer them here too for clarity. And the same applies to the docs.

Reply:
It is a test - shouldn't we test all aspects? As far as I can see, the docs use `hex""`